docs/frontend-api/pagination.md

```diff
@@ -0,0 +1,40 @@
+# Pagination
+
+Shopsys Framework Frontend API uses pagination inspired by [Relay cursor connection](https://facebook.github.io/relay/graphql/connections.htm)
+
+Every paginated entity uses connection with information about current page and paginated entities.
+For more information please see [the specification](https://facebook.github.io/relay/graphql/connections.htm)
+
+## Usage
+
+If you want to get eg. paginated products you can use `products` query.
+
+```json
+{
+  products{}
+}
+```
+
+This query will return a `connection` object which consists of `pageInfo` and `edges`.
+
+`pageInfo` is object that represents information about current page of pagination that you are on.
+
+`edges` are array of objects that implements `EdgeInterface` and represent the products that you get.
```

Comment:
this part of documentation not longer true :smile:

Reply:
Can you please update all documentation? You have example with old ProductEdge.

Reply:
@pesektomas